### PR TITLE
Adds Danger

### PR DESCRIPTION
**Pull request by vegaro**
- Adds Danger
- Updates `revenuecat/sdks-common-config` to use the common job

[CSDK-410]

[CSDK-410]: https://revenuecats.atlassian.net/browse/CSDK-410?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+  supported_labels_in_pr = supported_types & github.pr_labels
+  no_supported_label = supported_labels_in_pr.empty?
+  if no_supported_label
+    fail("Label the PR using one of the change type labels: #{supported_types}")
+  end
+end
+
+fail_if_no_supported_label_found
+
```
Comment (tonidero) [inline Dangerfile:12]: Hmm this is weird: 
image
Usually that symbol should be above... Not sure what it means here 🤷‍♂️

File: .circleci/config.yml [hunk 2]
```diff
@@ -231,4 +231,8 @@ workflows:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
         - equal: [ "update-native-dependencies", << pipeline.schedule.name >> ]
     jobs:
-      - automatic-dependency-update 
\ No newline at end of file
+      - automatic-dependency-update 
+
+  danger:
+    jobs:
+      - revenuecat/danger
```
Comment (tonidero) [inline .circleci/config.yml:238]: Will this also run on branches without PRs? Not sure if we have that option enabled in this repo as well...

**Review by tonidero — APPROVED** [review]
Just a couple of questions but looks good!

File: .circleci/config.yml (hunk 2 above)
Comment (vegaro) [reply .circleci/config.yml:238]: Good point! I just enabled it :)

File: Dangerfile (hunk 1 above)
Comment (vegaro) [reply Dangerfile:12]: 🤔 I have no idea haha but I tried removing the last line and re-adding it and Git didn't detect any change so I am not sure

Comment (vegaro) [reply Dangerfile:12]: ok, it was an extra space lol

**Pull request merged**